Dogecoin/doge-price-gbp.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python
+# coding=utf-8
+#
+# <bitbar.title>Dogecoin Ticker (£GBP)</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>impshum</bitbar.author>
+# <bitbar.author.github>impshum</bitbar.author.github>
+# <bitbar.desc>Displays current Dogecoin price from Cryptonator plus a few quick links.</bitbar.desc>
+# <bitbar.image>http://imgur.com/sJHMlxc</bitbar.image>
```

Comment:
Could you change this to the actual image: `https://i.imgur.com/sJHMlxc.png` please :smile: